command/agent/cache/cachememdb/index.go

```diff
@@ -95,3 +89,20 @@ func validIndexName(indexName string) bool {
 	}
 	return true
 }
+
+type ContextInfo struct {
```

Comment:
It makes more sense to have this in lease cache than in cachememdb. Don't you think?

Reply:
cachememdb still needs this and package cache imports it, so it would cause a dep cycle.

Reply:
I see. 👍